images/base/files/usr/local/bin/entrypoint

```diff
@@ -50,12 +158,18 @@ fix_mount() {
     sync
   fi
 
-  echo 'INFO: remounting /sys read-only'
-  # systemd-in-a-container should have read only /sys
-  # https://systemd.io/CONTAINER_INTERFACE/
-  # however, we need other things from `docker run --privileged` ...
-  # and this flag also happens to make /sys rw, amongst other things
-  mount -o remount,ro /sys
+  if [[ -n "${userns}" ]]; then
+    systemctl disable udev.service
```

Comment:
should we just statically disable this in the base image?

Reply:
Do you mean we should disable udev even for rootful mode?

Reply:
yeah, we can do it at build-time in the base image right?
currently it's defacto ~disabled by ro /sys anyhow.

Reply:
opened a separate PR:  https://github.com/kubernetes-sigs/kind/pull/1948

Reply:
this is merged now 👍

Reply:
Thanks for merging!